assets/scss/_styles_project.scss

```
@@ -375,6 +375,10 @@ a:not([href]):not([class]):hover {
   display: block;
   text-rendering: optimizelegibility;
   background-clip: text, text;
+  @media (max-width:768px){
+    font-size: 4rem;
```

We should centre align the text here, 
```suggestion
    font-size: 4rem;
    text-align: center;
```